root/static/scripts/common/constants.js

```diff
@@ -17,9 +17,22 @@ export const EMPTY_PARTIAL_DATE: PartialDateT = Object.freeze({
   year: null,
 });
 
+type N_l_T = () => string;
```

Comment:
@reosarevok wrote in https://github.com/metabrainz/musicbrainz-server/commit/91b0577b7cb3869b9f69ee2c77be4281f02b3ca2#r103558776
> If we're doing this, put it somewhere where it's reusable, because we have several other places in the code with `() => string` types specifically for `N_l`.

That’s about commit https://github.com/metabrainz/musicbrainz-server/pull/2896/commits/8d530b8a7b2ea0781dbbc6479b85d366bd63be35 by @mwiencek.